### ISSUE TITLE
Add support for in-line openapi

### ISSUE THREAD

**Issue by Virus288** [gajus/eslint-plugin-jsdoc#1300]
## Motivation

I use package [Swagger-Jsdoc](https://www.npmjs.com/package/swagger-jsdoc) . This plugin seems to be complaining about 
```js
/ **
* @openapi
*/
```
Which is used by Swagger-jsdoc, to mark openapi syntax. 
![image](https://github.com/user-attachments/assets/5965b0ae-6a7f-4430-ba9d-d83beb5782ae)

I would love for this plugin to have option to ignore jsdoc, which includes `@openapi`.

Sample code: https://github.com/Monsters-RPG-game/Gateway/blob/dev/src/structure/modules/user/register/router.ts
Minimal eslint rules. related this issue
```json
{
         "jsdoc/check-tag-names": 1,
         "jsdoc/check-indentation": 1
}
```

## Current behavior

Currently plugin complains about jsdoc, which has `@openapi`

## Desired behavior

Add some way to ignore jsdoc, which include `@openapi`, or add a way to ignore jsdoc blocks, which include specific tag

## Alternatives considered

N/A

**Comment by brettz9** [comment]
For `check-tag-names`, there is [this option](https://github.com/gajus/eslint-plugin-jsdoc/blob/main/docs/rules/check-tag-names.md#definedtags).

For `check-indentation`, you can either disable the rule or set [this option](https://github.com/gajus/eslint-plugin-jsdoc/blob/main/docs/rules/check-indentation.md#user-content-check-indentation-options-excludetags).

For example:

```js
"jsdoc/check-tag-names": ["warn", {"definedTags": ["openapi"]}],
"jsdoc/check-indentation": ["warn", {"excludeTags": ["openapi"]}]
```

**Comment by brettz9** [comment]
Closing as that should resolve, but feel free to comment further as needed.

**Comment by Virus288** [comment]
Thanks. I couldn't find it
